src/common/styles/fonts.scss

```diff
@@ -79,6 +79,13 @@ $semiBoldFontFamily: make-font-family($semiBoldFonts) !default;
     line-height: 24px;
 }
 
+@mixin text-style-header-s {
```

Comment:
the name should probably be headers instead of header-s

Reply:
this style is mainly for the text font style, so I think it should be as it is, same for text-style-title-* and text-style-body-*